examples/tutorial/hello/go/README.md

```diff
@@ -1,66 +1,138 @@
-# Tutorial 1: Go Function w/ Input (3 minutes)
+# Go Function Hello World
 
-This example will show you how to test and deploy Go (Golang) code to Fn. It will also demonstrate passing data in through stdin.
+This example will shows you how to test and deploy Go code to Fn. It will also demonstrates passing JSON data to your function through `stdin`.
 
-### First, run the following commands:
+This tutorial assumes you have installed Docker, Fn server, and Fn CLI. See the [Fn Quickstart](https://github.com/fnproject/fn) for installation steps.
+
+## Start Fn Server
+
+Start up the Fn server so we can deploy our function.
 
 ```sh
-# Initialize your function creating a func.yaml file
-fn init --name hello-go
+fn start
+```
 
-# Test your function. This will run inside a container exactly how it will on the server
-fn run
+The command starts Fn in single server mode using an embedded database and message queue. You can find all the
+configuration options [here](docs/operating/options.md). If you are on Windows, check [here](docs/operating/windows.md).
 
-# Now try with an input
-cat sample.payload.json | fn run
+## Create your Function 
 
-# Deploy your functions to the Fn server (default localhost:8080)
-# This will create a route to your function as well
-fn deploy --app myapp
-```
+1. Create an empty directory called `hello` and `cd` into it.
```

Comment:
worth saying that it's a cold function, for "hot" refer to FDK-go

Reply:
Hey Denis. So the purpose of these hello tutorials is to get from nothing to something as quickly as possible. So I don't think a hot v cold discussion is needed for that. But if you think its really really important I can put something in.